packages/packem-plugins/__tests__/unit/plugins/wasm/parse.test.ts

```
@@ -0,0 +1,161 @@
+    });
+
+    it("should keep a leading BOM in an export name rather than stripping it", () => {
+        expect.assertions(2);
+
+        // A name is an arbitrary UTF-8 string, so U+FEFF is data, not an encoding marker.
+        const bom = [0xef, 0xbb, 0xbf];
+        const exportSection = [0x07, 0x0a, 0x01, 0x06, ...bom, 0x61, 0x64, 0x64, 0x00, 0x00];
+        const withBom = Uint8Array.from([...ADD.subarray(0, 21), ...exportSection, ...ADD.subarray(30)]);
+
+        expect(() => new WebAssembly.Module(withBom)).not.toThrow();
+        expect(parseWasmModuleShape(withBom).exports).toStrictEqual([{ kind: "function", name: "\uFEFFadd" }]);
```

ESLint

[error] Prefer Unicode code point escapes.

(unicorn/prefer-unicode-code-point-escapes)